lib/install/management/dispatcher.go

```diff
@@ -218,6 +220,65 @@ func (d *Dispatcher) CollectDiagnosticLogs() {
 	}
 }
 
+// opManager creates a new operation manager.
```

Comment:
This comment is useless - please describe what an opManager is/does/why if you're going to have a comment here.